plugins/modules/proxmox_backup.py

```diff
@@ -110,19 +103,14 @@
   protected:
     description:
       - Marks backups as protected.
-      - >
-        "Might fail, when the PBS backend has verify enabled
-         due to this bug: U(https://bugzilla.proxmox.com/show_bug.cgi?id=4289)"
+      - '"Might fail, when the PBS backend has verify enabled due to this bug: U(https://bugzilla.proxmox.com/show_bug.cgi?id=4289)".'
     type: bool
   retention:
     description:
-      - >
-        Use custom retention options instead of those from the default cluster
-        configuration (which is usually V("keep-all")).
+      - Use custom retention options instead of those from the default cluster configuration (which is usually V("keep-all")).
       - Always requires Datastore.Allocate permission at the storage endpoint.
-      - >
-        Specifying a retention time other than V(keep-all=1) might trigger pruning on the datastore,
-        if an existing backup should be deleted target due to your specified timeframe.
+      - Specifying a retention time other than V(keep-all=1) might trigger pruning on the datastore, if an existing backup should be deleted target
```

Comment:
`target` can be deleted here. 

My original sentence was something like `should be a deletion target` and one of my changes forgot to remove this word.